packaging/pem.changes

```diff
@@ -1,3 +1,9 @@
+* 2016-07-27 Simo Kuusela <simo.kuusela@intel.com> 0.1.11
+  - Make PEM compatible with both Python 2 and Python 3.
+  - In src/emulator.py/_transmit_with_retry, timeout value
+    for _phys.receive_message had to be increased, because
+    with Python 3 it didn't work that well with 200ms.
+
```

Comment:
@skuusela apologies, I was not sufficiently clear, this looks good, but could you please add also a comment in the code? The release notes is not exactly the first place where one would look.